### DISCUSSION TITLE
Is it possible to get QR code from the matter.js import ?

### DISCUSSION THREAD

**Discussion by MatthieuCa** [matter-js/matter.js#1167] [Q&A] [answered]
Hello everyone,

It's me again...

I have a quick question: My app has a frontend part, and in this frontend, I want to display the QR code that Matter creates for us. Is there a function to get the QR code and return it to the frontend instead of displaying it in the console?

**Comment by Apollon77** [comment, marked as answer]
Yes sure ... see https://github.com/project-chip/matter.js/blob/67d747f2b877ff439514a6a9a48b55669fbb4b6f/packages/matter-node.js-examples/src/examples/DeviceNodeFull.ts#L396-L401
